airflow/cli/commands/connection_command.py

```diff
@@ -26,6 +26,8 @@
 from airflow.models import Connection
 from airflow.utils import cli as cli_utils
 from airflow.utils.session import create_session
+from airflow.secrets.local_filesystem import load_connections
```

Comment:
I think we should move these functions to another module in order not to create dependencies between CLI and `airflow.secrets.local_filesystem` package.

Reply:
That makes sense, can you suggest where should I move it?